tokio/src/io/util/vec_with_initialized.rs

```diff
@@ -56,7 +68,7 @@ where
         self.vec.as_mut().is_empty()
     }
 
-    pub(crate) fn get_read_buf<'a>(&'a mut self) -> ReadBuf<'a> {
+    pub(crate) fn get_read_buf(&mut self) -> ReadBuf<'_> {
```

Comment:
Can you undo this change? I'd like to keep the lifetime inside the `from_raw_parts_mut` call.

Reply:
Sure.  This was a bit of a gratuitous change in passing 😄 .  May I ask why you prefer the explicit `'a` though?

Reply:
It's mostly that `std::slice::from_raw_parts_mut` detaches the lifetime of the resulting slice from the owner, and I try to avoid that as much as possible.

Reply:
Right, `from_raw_parts_mut` uses an "unbounded" lifetime param.  However, inference should select the proper output lifetime when using the `'_` wildcard.

But, I reverted this part.